README.md

```diff
@@ -24,6 +24,7 @@ On top of standard Appium requirements Mac2 driver also expects the following pr
 - macOS 10.15 or later
     - macOS 11.3 or later and less than macOS 12, and Xcode 12 combination does not work as the build failure
 - Xcode 12 or later should be installed
+    - `xcode-select` should set for `Xcode.app/Contents/Developer` instead of `/Library/Developer/CommandLineTools` to run `xcodebuild` commands
```

Comment:
`xcode-select` should be pointing to `<full_path_to_xcode_app>/Contents/Developer` developer directory ....

Reply:
@KazuCocoa Would you like to update the comment?

Reply:
Sorry, i missed this